src/hooks/useSort.ts

```diff
@@ -0,0 +1,66 @@
+import { useMemo, useState } from 'react'
+import { ascend, descend, pick, pipe, prop, sortWith } from 'ramda'
+
+const sortDirections = {
+  ASC: ascend,
+  DESC: descend
+}
+
+const defaultInitial = {
+  sortBy: undefined,
+  sortDirection: undefined
+}
+
+type DefaultSorter = (...args: any[]) => any[]
+
+interface Sort {
+  sortBy: string | undefined
+  sortDirection: 'DESC' | 'ASC' | undefined
```

Comment:
@dannytce I don't think it could... but in this case, I'll ask you to hold for #40 and #50, which intend to fix multiple TypeScript issues... I'm not experienced in TypeScript so far, but on those tasks I'll probably learn enough to improve some parts and fix minor issues such as this one.